main/document/add_link.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * This script allows to add cloud file links to the document structure
+ * @package chamilo.document
+ */
+/**
+ * Code
+ */
+ 
+// Including the global initialization file
+require_once '../inc/global.inc.php';
+ 
+// Including additional libraries
+require_once '../inc/lib/document.lib.php';
+require_once '../inc/lib/urlUtils.lib.php';
+
+$fileLinkEnabled = api_get_configuration_value('enable_add_file_link');
+
+if (!$fileLinkEnabled) {
+    api_not_allowed(true);
+}
+ 
+$course_info = api_get_course_info();
+ 
+if (empty($course_info)) {
+    api_not_allowed(true);
+}
+ 
+$document_data  = DocumentManager::get_document_data_by_id($_REQUEST['id'], api_get_course_id(), true);
+if (empty($document_data)) {
+    $document_id  = $parent_id =  0;
+    $path = '/';
+} else {
+    if ($document_data['filetype'] == 'folder') {
+        $document_id    = $document_data['id'];
+        $path           = $document_data['path']."/";

```

Comment:
If you use single quotes for strings on line 33 (which is kind of better for micro-optimization purposes), use single quotes here too :-)